pyzeebe/channel/__init__.py

```diff
@@ -1,3 +1,7 @@
-from pyzeebe.channel.camunda_cloud_channel import create_camunda_cloud_channel
+# from pyzeebe.channel.camunda_cloud_channel import create_camunda_cloud_channel # FIXME: could be removed
```

Comment:
I think this can be removed, the new implementation also supports Camunda Cloud.